src/RestrictedPython/README.rst

```diff
@@ -67,26 +73,30 @@ Specifically:
    ``RestrictedPython.Guards.safe_builtins``.
 
 To help illustrate how this works under the covers, here's an example
```

Comment:
please do not break documentation lines wthin sentences.

Reply:
line breaks within sentences are in there all over the place. What's the reason for this policy?